packages/beacon-node/package.json

```diff
@@ -76,10 +76,10 @@
     "lint": "eslint --color --ext .ts src/ test/",
     "lint:fix": "yarn run lint --fix",
     "test": "yarn test:unit && yarn test:e2e",
-    "test:unit:minimal": "vitest --run --segfaultRetry 3 --dir test/unit/",
-    "test:unit:mainnet": "LODESTAR_PRESET=mainnet vitest --run --dir test/unit-mainnet",
-    "test:unit": "wrapper() { yarn test:unit:minimal $@ && yarn test:unit:mainnet $@; }; wrapper",
```

Comment:
This wrapper was just added recently by @jeluard to allow forwarding vitest flags like `-t` to all executed commands
- https://github.com/ChainSafe/lodestar/pull/6249